pkgs/development/compilers/julia/1.5.nix

```diff
@@ -12,7 +12,7 @@
 , CoreServices, ApplicationServices
 }:
 
-assert (!blas.isILP64) && (!lapack.isILP64);
+# assert (!blas.isILP64) && (!lapack.isILP64);
```

Comment:
Hm, so you could make `useSystemBLAS` an option and assert either that the inputs are good or that they are ignored. Default is based on the platform.

Do we not have a proper BLAS build on Darwin, or is it just broken?

Reply:
Sorry to have not replied for so long - I will need to investigate further - I have a feeling that one always use openblas on darwin